File: solutions/minTimeToVisitAllPoints.c
#include <stdio.h>
#include <stdlib.h>
//commit
int max(int a, int b) {
    return (a > b) ? a : b;
}

int minTimeToVisitAllPoints(int points[][2], int size) {
    int time = 0;
    for (int i = 1; i < size; i++) {
        int dx = abs(points[i][0] - points[i - 1][0]);
        int dy = abs(points[i][1] - points[i - 1][1]);
        time += max(dx, dy);
    }
    return time;
}

File: solutions/min_time_to_visit_all_points.py
#commit
Class Solution:
    def min_time_to_visit_all_points(points):
        time = 0
        for i in range(1, len(points)):
            dx = abs(points[i][0] - points[i - 1][0])
            dy = abs(points[i][1] - points[i - 1][1])
            time += max(dx, dy)
        return time
